tokio-util/src/time/wheel/level.rs

```diff
@@ -51,20 +46,17 @@ impl<T: Stack> Level<T> {
     pub(crate) fn next_expiration(&self, now: u64) -> Option<Expiration> {
         // Use the `occupied` bit field to get the index of the next slot that
         // needs to be processed.
-        let slot = match self.next_occupied_slot(now) {
-            Some(slot) => slot,
-            None => return None,
-        };
+        let slot = self.next_occupied_slot(now)?; // Replaced match with `?`
```

Comment:
This comment is not useful. Future readers of this code will just
```
let slot = self.next_occupied_slot(now)?; // Replaced match with `?`
```
without seeing the match. What are you trying to tell those future readers?